script.js

```diff
@@ -358,3 +358,26 @@ desktopProjects.forEach(element => {
       }
   }
 })
+
+//form validation starts
+
+const contactForm = document.querySelector('.contact-form');
+const contactError = document.querySelector('.contact-error');
+const emailInput = document.querySelector('#email');
+
+
+  function submitForm(event){
+  if (emailInput.value !== emailInput.value.toLowerCase()){
+    contactError.style.display = 'block';
+    contactError.style.backgroundColor = 'red';
+    contactError.textContent = "Email address should be lowercase";
+    event.preventDefault();
+
+  }
+
+  // else {
+  //   contactError.style.display = 'block';
+  //   contactError.style.backgroundColor = 'green';
+  //   contactError.textContent = "Email addressis in lowercase";
+  // };
```

Comment:
[OPTIONAL]

- [ ] Kindly make sure you delete the commented lines of code in your code base for making it clear and concise